pycheribuild/projects/disk_image.py

```diff
@@ -642,6 +643,21 @@ def make_efi_partition(self, efi_partition: Path):
                 self.run_cmd(mtools_bin / "mdir", "-i", efi_partition, "-/", "::")
                 # self.run_cmd(mtools_bin / "mdu", "-i", efi_partition, "-a", "::")
 
+    def make_rootfs_tarball(self):
+        # write out the manifest file:
+        self.mtree.write(self.manifest_file, pretend=self.config.pretend)
+        try:
+            bsdtar_path = shutil.which(str(self.bsdtar_cmd))
+            if not bsdtar_path:
+                bsdtar_path = str(self.bsdtar_cmd)
+            self.run_cmd([bsdtar_path, "cf", self.disk_image_path, "@"+str(self.manifest_file)], cwd=self.rootfs_dir)
```

Comment:
Is it worth compressing? Maybe as an option?

Reply:
Ah ha, `bsdtar` has a `-a` option.  How cute, let's go with that.

Reply:
> if it is unknown suffix or no suffix, creates a new archive with restricted pax format and bzip2 compression.

This seems undesirable...

Reply:
Though that seems inaccurate in my testing, where it does what you'd actually want, i.e. leave it uncompressed

Reply:
So yeah let's go with -a :)